spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitManagementTemplate.java

```diff
@@ -70,6 +70,20 @@ public RabbitManagementTemplate(Client rabbitClient) {
 
 	/**
 	 * Construct a template using the supplied uri.
+	 * @param uri the uri - must include user info, e.g.
+	 * "http://guest:guest@localhost:15672/api/".
```

Comment:
`@since 2.0`? 
And can we use a new `Client(String uri)` ctor already ?

Just let me know and I'll fix on merge